packages/graphql-language-service-server/src/startServer.ts

```diff
@@ -46,10 +47,13 @@ type Options = {
   // port for the LSP server to run on
   port?: number;
   // socket, streams, or node (ipc). if socket, port is required
-  method?: 'socket' | 'stream' | 'node';
```

Comment:
This looks incorrect. I think this should be `'socket' | 'stream' | 'node'`

Reply:
Fixed via https://github.com/graphql/graphiql/pull/1118/commits/f6d8a564708bdc7682d2c76248df15cc1f5ae69f